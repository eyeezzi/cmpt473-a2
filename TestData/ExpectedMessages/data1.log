Some error about records having different field counts. See RFC 4180 spec 4