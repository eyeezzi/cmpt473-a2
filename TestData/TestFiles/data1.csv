firstname, lastname, city
Donald, Knuth, Wisconsin
Ada, Lovelace, London