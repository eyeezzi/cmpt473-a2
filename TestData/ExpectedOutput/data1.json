[{" city":" Wisconsin"," lastname":" Knuth","firstname":"Donald"},{" city":" London"," lastname":" Lovelace","firstname":"Ada"}]